src/WebJobs.Extensions/Extensions/Timers/Scheduling/CronSchedule.cs

```diff
@@ -32,13 +61,133 @@ internal CrontabSchedule InnerSchedule
             }
         }
 
+        internal bool IsInterval { get; private set; }
+
         /// <inheritdoc/>
+        [Obsolete("This property is obsolete and will be removed in a future version.")]
         public override bool AdjustForDST => true;
 
-        /// <inheritdoc/>
+        /// <inheritdoc/>        
         public override DateTime GetNextOccurrence(DateTime now)
         {
-            return _cronSchedule.GetNextOccurrence(now);
+            // Note: TimeZoneInfo.Local is mocked in tests
+            return GetNextOccurrence(new DateTimeOffset(now)).LocalDateTime;
+        }
+
+        private DateTimeOffset GetNextOccurrence(DateTimeOffset now)
+        {
+            // The cron library only supports DateTime and does not take TimeZones into account. We need to 
```

Comment:
Looks like there's an ncrontab version 3.3.3 with bug fixes that we might consider taking. I only checked to see if there was some updated version with TimeZone support :)

Reply:
Yeah, I checked too :-)

Good point, I'll pull it in.